crates/katana-rpc/src/katana/mod.rs

```diff
@@ -23,4 +23,22 @@ impl<S: Sequencer + Send + Sync + 'static> KatanaApiServer for KatanaRpc<S> {
         self.sequencer.generate_new_block().await;
         Ok(())
     }
+
+    async fn block_timestamp(&self) -> Result<u64, Error> {
```

Comment:
i think this would be more appropriate to call it `pending_block_timestamp` as the block context is updated only when a pending block is created.